README.md

```diff
@@ -122,4 +122,13 @@ It sends an event to the uinput device.
 * *code* `Number`
 * *callback* `Function`
 
-Wrapper over send_event to simulate key presses and mouse clicks.
\ No newline at end of file
+Wrapper over send_event to simulate key presses and mouse clicks.
+
+### emit_combo(stream, code, callback)
+
+* *stream* `WritableStream`
+* *code* `Array with shift or alt and number or letter`
+* *callback* `Function`
```

Comment:
I think adding a call to `emit_combo` in the example from the README and [here](https://github.com/santigimeno/node-uinput/blob/master/examples/example.js) would make things clearer. Thanks!